web.py

```diff
@@ -79,12 +96,12 @@ def upload():
             
             #g = Github("fe45e3a06969900ef870e3a49adcc051e4172482")
             #g = Github("a0740516bc41f331e1dfc84ef64cedadd8d35c0e")
-            g = Github("76009c34088cb69d7392fc3a6baa963ab0bd4e8d ")
+            #g = Github("fa0dfb5e6103c9bf91cbbcc22e3c1aa5708cbdb4")
             #For app
             #Current pygithub version: 1.40, flask 1.0.2
             #g = Github("v1.b6634b8e67d274dfe131b07b9dd3494cd2446d12")
             #Get repository
-            repo = g.get_user().get_repo('woaq')#Need to change repository to main repository
+            #repo = g.get_user().get_repo('woaq')#Need to change repository to main repository
             repo.create_file('_posts/'+markDownFile,"Added a new air quality markdown data file on "+str(now)+"date.",mD,branch='gh-pages')
             #Saves file
             filename = finalFile
```

Comment:
Avoid names like ```finalFile``` that declare a position in a process. 
1) That process might change
2) I have no idea why it's in that position
3) I may not understand the process and telling me a position doesn't help clarify how the process works

Instead, use a name that describes its properties. ie) ```gps_aq_synced_file```